ext_mongodb.php

```diff
@@ -620,6 +620,78 @@ class BulkWriteException extends WriteException {}
 /* {{{ BSON and Serialization Classes */
 namespace MongoDB\BSON;
 
+/* {{{ Interfaces */
+interface TypeWrapper
+{
+	static public function createFromBSONType(\MongoDB\BSON\Type $type) : \MongoDB\BSON\TypeWrapper;
+	public function toBSONType() : \MongoDB\BSON\Type;
+}
+
+interface BinaryInterface
+{
+	public function __construct(string $data, int $type);
```

Comment:
It's for the benefit of application code that would like to accept such values. If someone is going to use a a custom MyObjectID class and have it masquerade as an ObjectID, application code would want an ObjectIDInterface to hint against.